CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+v0.29.0
```

Comment:
Need to bump this to `v0.30.0` as another PR just got merged which was 29.

Reply:
🥈